tests/test_cli.py

```diff
@@ -1240,6 +1240,28 @@ def test_simulate_long_args(self):
         assert args.random_seed == 123
         assert args.compress
 
+    def test_mutate_default_values(self):
+        parser = cli.get_msp_parser()
+        cmd = "mutate"
+        args = parser.parse_args([cmd, "out.trees", "out2.trees"])
+        assert args.tree_sequence == "out.trees"
+        assert args.output_tree_sequence == "out2.trees"
+        assert args.mutation_rate >= 0
+        assert args.random_seed is None
+        assert args.keep in (True, False)
```

Comment:
We should take a position here --- the default should be True

Reply:
Are you sure? The current default is definitely False. Changing it to true would mean making "--keep" a no-op and introducing a new flag to turn off keeping -- i.e something like "--discard" or "--no-keep"

Reply:
Sorry, I meant --discrete